config/database.yml

```diff
@@ -23,21 +23,21 @@ default: &default
 
 development:
   <<: *default
-  database: nuansacp_development
+  database: postgres
 
   # The specified database role being used to connect to postgres.
   # To create additional roles in postgres see `$ createuser --help`.
   # When left blank, postgres will use the default role. This is
   # the same name as the operating system user that initialized the database.
-  #username: nuansacp
+  username: postgres
 
   # The password associated with the postgres role (username).
-  #password:
+  password: 150101
 
   # Connect on a TCP socket. Omitted by default since the client uses a
   # domain socket that doesn't need configuration. Windows does not have
   # domain sockets, so uncomment these lines.
-  #host: localhost
+  host: localhost
```

Comment:
You can change this in your local files, but it would be better for you to not commit change for this part.